src/Transport.ts

```diff
@@ -331,320 +338,383 @@ export default class Transport {
   async request<TResponse = unknown, TContext = any> (params: TransportRequestParams, options?: TransportRequestOptionsWithMeta): Promise<TransportResult<TResponse, TContext>>
   async request<TResponse = unknown> (params: TransportRequestParams, options?: TransportRequestOptions): Promise<TResponse>
   async request (params: TransportRequestParams, options: TransportRequestOptions = {}): Promise<any> {
-    const connectionParams: ConnectionRequestParams = {
-      method: params.method,
-      path: params.path
-    }
+    const otelTracer = opentelemetry.trace.getTracer('elastic-transport')
```

Comment:
To the degree that `request(...)` is a hot path (hotter than the Transport constructor), would it be reasonable to move `otelTracer` to an internal property of `class Transport` and move `getTracer(...)` to the constructor?

Reply:
Fixed in 96cb20ff4f5e81a6786288d1aa3b12ed36a59864.